core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -453,19 +453,29 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
             val entityTypeOpt = workflowRecord.workflowEntityId.flatMap(entitiesById.get).map(_.entityType)
             OutputExpression.build(outputExprStr, output, entityTypeOpt)
         }
-      }
+      }.toSeq
 
       if (parsedExpressions.forall(_.isSuccess)) {
-        val boundExpressions = parsedExpressions.collect { case Success(boe @ BoundOutputExpression(target, name, attr)) => boe }
+        val boundExpressions: Seq[BoundOutputExpression] = parsedExpressions.collect { case Success(boe @ BoundOutputExpression(target, name, attr)) => boe }
         val updates = updateEntityAndWorkspace(workflowRecord.workflowEntityId.map(id => Some(entitiesById(id))).getOrElse(None), workspace, boundExpressions)
+
+        val (optEntityUpdates, optWs) = updates
+        optEntityUpdates foreach { update: WorkflowEntityUpdate =>
+          logger.debug(s"Updating ${update.upserts.size} attributes for entity ${update.entityRef.entityName} of type ${update.entityRef.entityType} in ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}. First 100: ${update.upserts.take(100)}")
+        }
+        optWs foreach { workspace: Workspace =>
+          logger.debug(s"Updating ${workspace.attributes.size} workspace attributes in ${submissionId.toString}/${workflowRecord.externalId.getOrElse("MISSING_WORKFLOW")}. First 100: ${workspace.attributes}")
```

Comment:
There's no `.take(100)` here. Maybe add one, or else remove the `"First 100:"` in the comment?

Reply:
Nice find! Fixing.